chef/cookbooks/nova_dashboard/recipes/server.rb

```diff
@@ -281,9 +281,7 @@
     :password_validator_help_text => node[:nova_dashboard][:password_validator][:help_text],
     :site_branding => node[:nova_dashboard][:site_branding],
     :site_branding_link => node[:nova_dashboard][:site_branding_link],
-    :neutron_networking_plugin => neutron_networking_plugin,
-    :neutron_networking_mode => neutron_networking_mode,
-    :neutron_use_ml2 => neutron_use_ml2,
+    :neutron_ml2_type_drivers => neutron[:neutron][:ml2_type_drivers],
```

Comment:
> s/neutron[:neutron][:ml2_type_drivers]/neutron_ml2_type_drivers/

Done in https://github.com/toabctl/barclamp-nova_dashboard/commit/3d336438a2553fc99ac3284269a3ac680583e4b3